tools/ci-build/create-local-build-image

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
not opposed to all these bash scripts necessarily but we should definitely start using shellcheck